src/Numerics/Exceptions.cs

```diff
@@ -20,7 +20,7 @@ public NonConvergenceException(string message) : base(message)
         public NonConvergenceException(string message, Exception innerException) : base(message, innerException)
         {
         }
-#if !PORTABLE
+#if !PORTABLE && !NOSERIALIZATION
```

Comment:
We should consider removing the PORTABLE symbol all together in favor of feature flags.